google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/project/ProjectSelector.java

```diff
@@ -91,14 +87,9 @@ public ProjectSelector(@Nullable Project ideProject) {
                     });
           }
         };
-    messageBusConnections =
-        Stream.of(ProjectManager.getInstance().getOpenProjects())
-            .map(p -> p.getMessageBus().connect())
-            .collect(Collectors.toSet());
-    messageBusConnections.forEach(
-        connection ->
-            connection.subscribe(
-                GoogleLoginListener.GOOGLE_LOGIN_LISTENER_TOPIC, googleLoginListener));
+    messageBusConnection = ApplicationManager.getApplication().getMessageBus().connect();
```

Comment:
how will this behave if you have multiple projects opened?

Reply:
It gets called for all listeners regardless of how many projects are opened, or no projects are opened at all, so should be good.

Reply:
just so I understand: since we currently can't be signed into different accounts in different projects (right?) then this is ok. since a logout in one project should be a logout in all projects

Reply:
oh. nm. I see now

Reply:
Right, it's completely generic event, i.e. no correlation between IDE projects and login state.